docker/Dockerfile.tcf-dev

```diff
@@ -86,33 +86,34 @@ RUN wget https://download.01.org/intel-sgx/linux-2.3.1/ubuntu18.04/sgx_linux_x64
  && rm sgx_linux_x64_sdk_2.3.101.46683.bin \
  && echo ". /opt/intel/sgxsdk/environment" >> /etc/environment
 
-# ("Untrusted") OpenSSL
 WORKDIR /tmp
-RUN wget https://www.openssl.org/source/openssl-1.1.1d.tar.gz \
- && tar -zxvf openssl-1.1.1d.tar.gz \
- && cd openssl-1.1.1d/ \
+# py-solc 3.2.0 requires an older version of the solc compiler, 0.4.25
+RUN mkdir -p $HOME/.py-solc/solc-v0.4.25/bin \
+  && curl -LsS https://github.com/ethereum/solidity/releases/download/v0.4.25/solc-static-linux -o /root/.py-solc/solc-v0.4.25/bin/solc \
+  && chmod 0755 /root/.py-solc/solc-v0.4.25/bin/solc
+
+# ("Untrusted") OpenSSL
+RUN OPENSSL_VER=1.1.1d \
+ && wget https://www.openssl.org/source/openssl-$OPENSSL_VER.tar.gz \
+ && tar -zxf openssl-$OPENSSL_VER.tar.gz \
+ && cd openssl-$OPENSSL_VER/ \
  && ./config \
  && THREADS=8 \
  && make -j$THREADS \
  && make test \
  && make install -j$THREADS \
  && ldconfig \
  && cd .. \
- && rm -rf openssl-1.1.1d
-# Note: we do _not_ delete openssl-1.1.1d.tar.gz as we re-use it below ..
-
-# py-solc 3.2.0 requires an older version of the solc compiler, 0.4.25
-RUN mkdir -p $HOME/.py-solc/solc-v0.4.25/bin \
-  && curl -LsS https://github.com/ethereum/solidity/releases/download/v0.4.25/solc-static-linux -o /root/.py-solc/solc-v0.4.25/bin/solc \
-  && chmod 0755 /root/.py-solc/solc-v0.4.25/bin/solc
+ && rm -rf openssl-$OPENSSL_VER
+# Note: we do _not_ delete openssl-*.tar.gz as we re-use it below ..
```

Comment:
Can't we use openssl-$OPENSSL_VER.tar.gz

Reply:
No. This is removing the build directory, not the tar file. So `openssl-$OPENSSL_VER` (without `.tar.gz`) is correct. The tar file is needed below as mentioned in the comment on line 108:
`# Note: we do _not_ delete openssl-*.tar.gz as we re-use it below ..`

Reply:
ok. instead of * can we use $OPENSSL_VER

delete openssl-*.tar.gz -> delete openssl-$OPENSSL_VER.tar.gz

Reply:
Done. I had to combine the `RUN` statements so `$OPENSSL_VER` is still defined.